src/ExternalLink.tsx

```diff
@@ -1,15 +1,15 @@
-import React, { FunctionComponent, ReactNode } from 'react';
```

Comment:
I prefer explicit imports. Don't think it's necessary to change this.

Reply:
Eslint would complain before I changed this.